### ISSUE TITLE
Unable to open menu items with VoiceOver on

### ISSUE THREAD

**Issue by JuliannaR** [CovidShield/mobile#170]
**Accessibility**

**Scenario to reproduce**

When VO is on and you tap to open the menu within the Shield VO cannot scan the elements, you need to know exactly where to tap to find items like language, province change, etc.

**Expected result**

VO should be able to scan all elements when dragging a finger (gesture) on points on the screen.

**Bug or issue / WCAG**

2.5.1: Pointer Gestures
https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html

**Screenshots where available**
![Unable to open menu items VO 3](https://user-images.githubusercontent.com/35537308/86022719-bd77e280-b9f8-11ea-8477-6402d9602c56.png)
![Unable to open menu items with VO on 2](https://user-images.githubusercontent.com/35537308/86022742-c23c9680-b9f8-11ea-9873-b67c766e2b9d.jpg)
![unable to open menu items with voice over on](https://user-images.githubusercontent.com/35537308/86022751-c5d01d80-b9f8-11ea-81d7-e530471e37c6.jpg)

**Device + OS**

iPhone 8 | 13.5.1
